remote-config.jsonc

```diff
@@ -16,16 +16,16 @@
 			"interval": 20,
 			"messages": [
 				{
-					"message": "We’re actively looking for financial sponsors! https://github.com/sponsors/ddev"
+					"message": "You can make DDEV last by becoming a financial sponsor!\nhttps://github.com/sponsors/ddev"
 				},
 				{
-					"message": "A 403 from your project may mean the docroot is not specified or `index.php` is missing."
+					"message": "A 403 from your project may mean that\nthe docroot is not specified or `index.php` is missing."
```

Comment:
I don't like to introduce line breaks in the middle ot the text, console should take care about correct wrapping instead imho.